sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/automl_job.py

```diff
@@ -34,7 +34,28 @@
 
 
 class AutoMLJob(Job, JobIOMixin, AutoMLNodeIOMixin, ABC):
-    """AutoML job entity."""
+    """Initialize an AutoML job entity.
+
+    Constructor for an AutoMLJob.
```

Comment:
why replicate the comment from below here?

Reply:
Both the class and constructor must be documented.
__init__ docstring is not visible on the learning docs portal, but the docstring from init method is extracted on running `help(YourClassName. __init__)`. A good strategy to follow is to make your changes in the class docstring and then copy the changes to the __init__ method docstring.